js/app.js

```diff
@@ -40,87 +28,233 @@ var dogDuck = 0;
+//TODO make random not equal last 3 nor other 2
+  if (attempts) {
+    productImageParent.removechild(productImageParent.lastChild);
+    productImageParent.removechild(productImageParent.lastChild);
+    productImageParent.removechild(productImageParent.lastChild);
+  }
+  // for (var p = 0; p < 3; i++) {
+  //   productNames ;
+  // }
+  renderProductImage(productImage1);
+  renderProductImage(productImage2);
+  renderProductImage(productImage3);
+  // updateAnswer();
+  // updateAttempts();
+}
+setup();
+renderResponse();
+data();
+buttonLogic();
+
+
+productImageParent.addEventListener('render', function(render) {
   var display = event.target.getAttribute('id');
+  render(display);
   //same logic as below? Find a way to loop this to make code dry
-})
-productImagesParent.addEventListener('click', function(event){
+});
+
+
+productImageParent.addEventListener('click', function(event) {
+  if (attempts === maxAttempts) {
+    return;
+  }
+
   var answer = event.target.getAttribute('id');
-  console.log('id');
-  if (answer != []) {
-    alert('please click on image')
-  } if  (answer === [0]) {
-    console.log(babySweep +=  1);
+  attempts++;
+//for i statement - constructor
+  if (answer === [0]) {
+    babySweep++;
   } else if (answer === [1]) {
-    console.log(bag += 1);
+    bag++;
   } else if (answer === [2]) {
-    console.log(banana += 1);
+    banana++;
   } else if (answer === [3]) {
-    console.log(bathroom += 1);
+    bathroom++;
   } else if (answer === [4]) {
-    console.log(boots += 1);
+    boots++;
   } else if (answer === [5]) {
-    console.log(breakfast += 1);
+    breakfast++;
   } else if (answer === [6]) {
-    console.log(bubblegum += 1);
+    bubblegum++;
   } else if (answer === [7]) {
-    console.log(chair += 1);
+    chair++;
   } else if (answer === [8]) {
-    console.log(cthulhu += 1);
+    cthulhu++;
   } else if (answer === [9]) {
-    console.log(dogDuck += 1);
+    dogDuck++;
   } else if (answer === [10]) {
-    console.log(dragon += 1);
+    dragon++;
   } else if (answer === [11]) {
-    console.log(pen += 1);
+    pen++;
   } else if (answer === [12]) {
-    console.log(petSweep += 1);
+    petSweep++;
   } else if (answer === [13]) {
-    console.log(scissors += 1);
+    scissors++;
   } else if (answer === [14]) {
-    console.log(shark += 1);
+    shark++;
   } else if (answer === [15]) {
-    console.log(tauntaun += 1);
+    tauntaun++;
   } else if (answer === [16]) {
-    console.log(unicorn += 1);
+    unicorn++;
   } else if (answer === [17]) {
-    console.log(usb += 1);
+    usb++;
   } else if (answer === [18]) {
-    console.log(waterCan += 1);
+    waterCan++;
   } else if (answer === [19]) {
-    console.log(wineGlass += 1);
+    wineGlass++;
   } else {
-      alert('please click on image')
-  };
+    alert('please click on image');
+  }
+
+  if (attempts === maxAttempts) {
+    chart();
+  }
+});
+
+function ProductName(name, click, shown) {
```

Comment:
If click and shown start at zero, do they need to be part of the parameters for your constructor?